img/tn-chartbeat.svg

```diff
@@ -1 +1,17 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 83 83"><defs><style>.cls-1{fill:#a8a8ab;}</style></defs><title>chartbeat</title><g id="Layer_2" data-name="Layer 2"><g id="Layer_1-2" data-name="Layer 1"><path class="cls-1" d="M41.5,0A41.5,41.5,0,1,0,83,41.5,41.5,41.5,0,0,0,41.5,0ZM16.25,23.16H67.43l-9.08,8H16.25Zm-9.6,19.6v-8h47.8l-9.08,8Zm11.08,3.66H40.84l-9.07,8h-14Zm19,20.6-.07,0a1.63,1.63,0,0,1-.17.2c-1,1-3,.64-4.49-.82S30.1,63,31.09,62a1.54,1.54,0,0,1,.19-.17.56.56,0,0,1,0-.08l40.9-38.2A1.24,1.24,0,1,1,74,25.26Z"/></g></g></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="17px" height="12px" viewBox="0 0 17 12" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: sketchtool 47.1 (45422) - http://www.bohemiancoding.com/sketch -->
+    <title>434E1380-4044-4EBB-8F41-4CCE4DF01FEE</title>
+    <desc>Created with sketchtool.</desc>
+    <defs></defs>
+    <g id="V4--RC-Updates" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="ASSETS--Pill-Icons" transform="translate(-179.000000, -470.000000)" fill-rule="nonzero" fill="#FFFFFF">
+            <g id="chartbeat" transform="translate(179.000000, 470.000000)">
+                <polygon id="Shape" points="8.93916667 5.5675 3.10722222 5.5675 3.10722222 7.57916667 6.64888889 7.57916667"></polygon>
+                <polygon id="Shape" points="12.0605556 2.78138889 0 2.78138889 0 4.79305556 9.77027778 4.79305556"></polygon>
+                <polygon id="Shape" points="15.1819444 0 2.26666667 0 2.26666667 2.01166667 12.8916667 2.01166667"></polygon>
+                <path d="M16.7119444,0.155833333 C16.5888041,0.034625509 16.3911959,0.034625509 16.2680556,0.155833333 L5.94527778,9.79388889 L5.94527778,9.79388889 L5.89333333,9.83638889 C5.64777778,10.0913889 5.75166667,10.5966667 6.12944444,10.965 C6.50722222,11.3333333 7.0125,11.4372222 7.26277778,11.1727778 L7.30527778,11.1208333 L7.30527778,11.1208333 L16.7072222,0.580833333 C16.8157404,0.460625556 16.8177649,0.278422651 16.7119444,0.155833333 L16.7119444,0.155833333 Z" id="Shape"></path>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
Can you either manually remove all the noise added by Zeplin from these .svg files or run them thru an svg cleaner like svgo from your local command line? https://www.npmjs.com/package/svgo

There doesnt need to be comments, title, desc, or defs tags and it doesn't look like any of the id's on the <g> tags are beig used. Get rid of all of that.

Reply:
svgo did the trick